src/client/components/segment-bubble/segment-bubble.tsx

```diff
@@ -81,6 +83,24 @@ export class SegmentBubble extends React.Component<SegmentBubbleProps, SegmentBu
     }
   }
 
+  goToURL(label: string) {
+    var { urls, datum } = this.props;
+    var idx = 0;
+    if (urls.length !== 1) {
+      const nestIndex = (datum['__nest'] as number) - 1;
+      idx = nestIndex;
+    }
+    var url = urls[idx];
+
+    if (!url) url = defaultGoToUrl;
+    var replacedUrl = url.replace('%s', label);
+
+    window.open(
+      replacedUrl, '_blank'
+    );
```

Comment:
I think it is better to use an `<a>` element instead of doing it in jabascript, then people's shortcuts will work better